osu.Android/OsuGameActivity.cs

```diff
@@ -72,11 +73,23 @@ protected override void OnCreate(Bundle savedInstanceState)
             Debug.Assert(Resources?.DisplayMetrics != null);
 
             Point displaySize = new Point();
+#pragma warning disable 618 // GetSize is deprecated
             WindowManager.DefaultDisplay.GetSize(displaySize);
+#pragma warning restore 618
             float smallestWidthDp = Math.Min(displaySize.X, displaySize.Y) / Resources.DisplayMetrics.Density;
             bool isTablet = smallestWidthDp >= 600f;
 
             RequestedOrientation = DefaultOrientation = isTablet ? ScreenOrientation.FullUser : ScreenOrientation.SensorLandscape;
+
+            // Currently (SDK 6.0.200), BundleAssemblies is not runnable for net6-android.
+            // The assembly files are not available as files either after native AOT.
+            // Manually load them so that they can be loaded by RulesetStore.loadFromAppDomain.
+            // REMEMBER to fully uninstall previous version every time when investigating this!
+            // Don't forget osu.Game.Tests.Android too.
+            Assembly.Load("osu.Game.Rulesets.Osu");
+            Assembly.Load("osu.Game.Rulesets.Taiko");
+            Assembly.Load("osu.Game.Rulesets.Catch");
+            Assembly.Load("osu.Game.Rulesets.Mania");
```

Comment:
This should also be in ruleset specific test apps (probably load only the tested ruleset).

Reply:
I'm not pretty sure if it's actually required. Debug build doesn't require this, as assemblies can be loaded from file. Only OsuGame and RulesetStore would impact from this, ruleset specific tests explicitly creates the ruleset.

Reply:
Looks like you're right, builds and runs just fine, in debug and release.